src/parser/demonhunter/havoc/modules/talents/FelBarrage.js

```diff
@@ -21,19 +24,58 @@ class FelBarrage extends Analyzer{
       return;
     }
     this.addEventListener(Events.damage.by(SELECTED_PLAYER).spell(SPELLS.FEL_BARRAGE_DAMAGE), this.felBarrage);
+    this.addEventListener(Events.cast.by(SELECTED_PLAYER).spell(SPELLS.FEL_BARRAGE_TALENT), this.felBarrageCasts);
   }
 
   felBarrage(event) {
     this.damage += event.amount;
   }
 
+  felBarrageCasts(event) {
+    this.casts += 1;
+
+    const hasMetaBuff = this.selectedCombatant.hasBuff(SPELLS.METAMORPHOSIS_HAVOC_BUFF.id, event.timestamp);
+
+    if (!hasMetaBuff) {
+      this.badCasts += 1;
+    }
+  }
+
+  get suggestionThresholds() {
+    return {
+      actual: this.badCasts,
+      isGreaterThan: {
+        minor: 0,
+        average: 0,
+        major: 1,
+      },
+      style: 'number',
+    };
+  }
+
+  suggestions(when) {
+    when(this.suggestionThresholds)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest(<>Try to cast <SpellLink id={SPELLS.FEL_BARRAGE_TALENT.id} /> when you have meta up to maximize its damage.</>)
+          .icon(SPELLS.FEL_BARRAGE_TALENT.icon)
+          .actual(`${actual} bad casts`)
+          .recommended(`No bad casts is recommended.`);
+      });
+  }
+
   statistic(){
     return (
       <TalentStatisticBox
         talent={SPELLS.FEL_BARRAGE_TALENT.id}
         position={STATISTIC_ORDER.OPTIONAL(6)}
-        value={this.owner.formatItemDamageDone(this.damage)}
-        tooltip={`${formatThousands(this.damage)} Total damage`}
+        value={(<>
+          {this.badCasts} Bad casts<br />
+          {this.owner.formatItemDamageDone(this.damage)}
+        </>)}
+        tooltip={`
+          ${formatThousands(this.damage)} Total damage <br /><br />
```

Comment:
You can no longer use HTML inside template strings in tooltips. I'd also put the reason on top, since that's the more important info. Making:

```jsx
        tooltip={(
          <>
            A bad cast is casting Fel Barage without Meta up.<br /><br />

            {formatThousands(this.damage)} total damage
          </>
        )}
```